README.md

```diff
@@ -1 +1,104 @@
-# To-Do-List
\ No newline at end of file
+<a name="readme-top"></a>
+
+# To-Do List Project
+
+# 📗 Table of Contents
+
+- [📖 About the To-Do List Project](#about-project)
+- [🛠 Built With](#built-with)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgments)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 To-do List Project <a name="about-project"></a>
+
+**"To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **made with use of github workflow**
+- **made with use of linter checkers**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JS</li>
+  </ul>
+</details>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Coming Soon!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Portfolio presentation will be coming soon.
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need a Chrome, Safari or Mozilla browser
+
+### Setup <a name="setup"></a>
+
+Clone this repository to your desired folder: https://github.com/jossi30/To-Do-List.git
+
+### Install <a name="install"></a>
+
+git clone https://github.com/jossi30/To-Do-List.git
+
```

Comment:
- [OPTIONAL] To install the project, you should run the `npm install` command. This will install all the dependencies needed to run this project. 👍 😎